app/src/main/java/com/example/lhi06/residue/AlbumReviewActivity.java

```diff
@@ -20,11 +20,11 @@
 
 public class AlbumReviewActivity extends AppCompatActivity {
 
-    @BindView(R.id.textview_album_name_review) TextView albumName;
-    @BindView(R.id.textview_artist_name_review) TextView artistName;
-    @BindView(R.id.imageview_album_art_review) ImageView albumArtwork;
+    @BindView(R.id.textview_album_name_review) TextView albumNameTextView;
+    @BindView(R.id.textview_artist_name_review) TextView artistNameTextView;
+    @BindView(R.id.imageview_album_art_review) ImageView albumArtworkTextView;
     @BindView(R.id.ratingBar) RatingBar ratingBar;
-    @BindView(R.id.reviewText) EditText reviewText;
+    @BindView(R.id.reviewText) EditText reviewEditText;
```

Comment:
Take a look at how you named the other IDs and fix this one. Same for the `ratingBar`